### PR TITLE
TC 11-12

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
I'm not sure what to do with "The Supplement" part near line 960. Maybe it's okay, but I thought it looked a little funky.

File: journal/modernized/11-chapter-11.adoc [hunk 1]
```diff
@@ -476,7 +489,7 @@ and even now have destroyed more on the like occasion.
 The English going beyond the bounds of their agreement,
 eleven of them were killed by the Indians while we were travelling in Virginia.
 
-"`Sixth;
+_Sixth;_
```
Comment (Henderjay) [inline journal/modernized/11-chapter-11.adoc:493]: You forgot the [.numbered] here 👍🏻

**Review by Henderjay — CHANGES_REQUESTED** [review]
You just forgot one [.numbered] tag above "sixth" in the modernized. And when you get to the Supplement at the bottom, I think we should use [.old-style] again, so that the heading and subheading are centered. Also... the comment in brackets just above The Supplement, let's give that an [.offset]

**Comment by Henderjay** [comment]
looks good, merged.

**Pull request merged**